Cargo.toml

```diff
@@ -11,8 +11,13 @@ tracing = "0.1"
 tracing-subscriber = "0.2"
 tracing-futures = "0.2"
 tokio = { version = "1.5.0", features = ["full"] }
+toml = "0.5"
 futures = "0.3.14"
 byteorder = "1.4.3"
+samplerate = "0.2.4"
```

Comment:
I don't think most of these new dependencies are needed.

Reply:
Yeah, those were leftovers from the earlier try w\ figment.